### PR TITLE
Oy2 11266 - Add Notificaiton

### PR DESCRIPTION
**Pull request by jdinh8124**
## Purpose
Add a notification that can be used to show success, error, warning, and info messages.

#### Linked Issues to Close
https://qmacbis.atlassian.net/jira/software/c/projects/OY2/boards/208?modal=detail&selectedIssue=OY2-11266

## Approach
Used Chakra's built in notification component

Screen Shot 2021-12-10 at 8 40 00 AM

#### Pull Request Creator Checklist

- [x] This PR has an associated issue or issues.
- [x] The associated issue(s) are linked above.
- [x] This PR meets all acceptance criteria for those issues.
- [x] This PR and linked issue(s) are adequately documented
- [x] This PR and linked issues(s) are a complete description of the changeset; an individual or team should be able to understand the issue(s) and changes by reading through this PR and it's links, with no further interaction.
- [ ] Someone has been assigned this PR.
- [x] At least one person has been marked as reviewer on this PR.

#### Pull Request Reviewer/Assignee Checklist

- [ ] This PR has an associated issue or issues.
- [ ] The associated issue(s) are linked above.
- [ ] This PR meets all acceptance criteria for those issues.
- [ ] This PR and linked issue(s) are adequately documented
- [ ] This PR and linked issues(s) are a complete description of the changeset; an individual or team should be able to understand the issue(s) and changes by reading through this PR and it's links, with no further interaction.

### CODE REVIEW

File: services/ui-src/src/components/Notification/index.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+}
+
+export const Notification = ({
+  alertProps,
+  alertTitle,
+  alertDescription,
+  alertStatus,
+  close,
+}: NotificationProps) => {
+  let borderColor = "green.500";
```
Comment (hasanfar) [inline services/ui-src/src/components/Notification/index.tsx:18]: Could this color values be in an enum..

**Review by hasanfar — COMMENTED** [review]
Looks nice

**Review by abelzer212 — DISMISSED** [review]

File: services/ui-src/src/components/Notification/index.tsx (hunk 1 above)
Comment (benjaminpaige) [reply services/ui-src/src/components/Notification/index.tsx:18]: yeah maybe a borderColor enum would be cool

Comment (jdinh8124) [reply services/ui-src/src/components/Notification/index.tsx:18]: Absolutely

File: services/ui-src/src/views/DemoComponents/index.tsx [hunk 2]
```diff
@@ -406,6 +410,82 @@ const DemoComponentsForm = () => {
...
+            <CUI.HStack>
+              <ContainedButton
+                disabledStatus={showSuccessAlert}
+                buttonText={`Show Success Alert`}
+                buttonProps={{
+                  colorScheme: "green",
+                  textTransform: "capitalize",
+                }}
+                onClick={() => setSuccessAlert(true)}
+              />{" "}
```
Comment (benjaminpaige) [inline services/ui-src/src/views/DemoComponents/index.tsx:459]: i hate how prettier adds these spaces

**Review by benjaminpaige — APPROVED** [review]
yeah it looks really good. love it!

**Review by hasanfar — APPROVED** [review]

**Pull request merged**